source/tessellation/VoronoiMesh.cpp

```diff
@@ -998,7 +1002,7 @@ void VoronoiMesh::GetToTest(vector<vector<int> > &copied,vector<vector<int> > &t
 	for(int i=0;i<nsides;++i)
 		sort(copied[static_cast<size_t>(i)].begin(),copied[static_cast<size_t>(i)].end());
 	totest.resize(static_cast<size_t>(nsides));
-	int test=0;
```

Comment:
it is good practice to give an initial value

Reply:
If it is used

Reply:
if it is an used variable get rid of it

Reply:
The value is not used, the variable is